task.sql

```diff
@@ -38,3 +38,16 @@ CREATE TABLE ProductInventory (
     PRIMARY KEY (ID)
 );
 --rollback DROP TABLE ProductInventory;
+
+--changeset illia:5 labels:0.0.2
```

Comment:
There is an inconsistency in the naming convention of the changeset authors. Previous changesets use 'mate.acamemy', while this one uses 'illia'. It would be better to maintain a consistent naming convention for changeset authors.